scram/route_manager/views.py

```diff
@@ -24,8 +24,10 @@
 channel_layer = get_channel_layer()
 
 
-def home_page(request, prefilter=Entry.objects.all()):
+def home_page(request, prefilter=None):
```

Comment:
What does this change do?

Reply:
It's this check: https://docs.astral.sh/ruff/rules/function-call-in-default-argument/

When you have a function call as the default parameter, that call happens only when the function is defined. So, we're prefiltering to whatever Entries we have when we start Django.

The recommended approach is to pass None with each `home_page` function call, check to see if it's None, and then pull the current set of Entries if it's None.